frontend/src/scenes/web-analytics/webAnalyticsLogic.ts

```diff
@@ -837,9 +886,88 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
     windowValues({
         isGreaterThanMd: (window: Window) => window.innerWidth > 768,
     }),
+
+    actionToUrl(({ values }) => ({
+        setWebAnalyticsFilters: () => stateToUrl(values),
+        togglePropertyFilter: () => stateToUrl(values),
+        setDates: () => stateToUrl(values),
+        setInterval: () => stateToUrl(values),
+        setDeviceTab: () => stateToUrl(values),
+        setSourceTab: () => stateToUrl(values),
+        setGraphsTab: () => stateToUrl(values),
+        setPathTab: () => stateToUrl(values),
+        setGeographyTab: () => stateToUrl(values),
+    })),
```

Comment:
Here's a patter you can use if you want to clean this up a bit, yet keep the url logic contained:

```suggestion
    actionToUrl(({ values }) => { 
        function stateToUrl() {
            const urlParams = new URLSearchParams()
            // ...
        }
        return {
            setWebAnalyticsFilters: stateToUrl,
            togglePropertyFilter: stateToUrl,
            setDates: stateToUrl,
            setInterval: stateToUrl,
            setDeviceTab: stateToUrl,
            setSourceTab: stateToUrl,
            setGraphsTab: stateToUrl,
            setPathTab: stateToUrl,
            setGeographyTab: stateToUrl,
        }
    })),
```

This is just a nit at this point, definitely not blocking.